directory/tests/conftest.py

```diff
@@ -13,14 +13,19 @@
 
 
 @pytest.fixture(scope="session")
-def dd_environment():
+def dd_environment(instance):
     compose_file = os.path.join(common.HERE, "compose", "docker-compose.yml")
     with docker_run(
         compose_file=compose_file,
     ):
-        yield common.get_config_stubs(".")[0]
+        yield instance
```

Comment:
Let's not used the instance fixture here since it would require the `session` scope.

Reply:
So it is better to use `common.get_config_stubs(".")[0]` ? Or should we directly 
```python
yield {
      'directory': '.',
      'filegauges': False,
      'tags': ['optional:tag1']
}
```
?

Reply:
`common.get_config_stubs(".")[0]` is fine